### PR TITLE
feat(denoms): Add 2 simple tables with asset info

### PR DESCRIPTION
**Pull request by 0xFable**

### CODE REVIEW

File: gitbook/smart-contracts/assets /denoms.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Assets and related info
+
+## Migaloo on each zone
```
Comment (kerber0x) [inline gitbook/smart-contracts/assets /denoms.md]:
```suggestion
## Whale on each zone
```

Comment (kerber0x) [reply gitbook/smart-contracts/assets /denoms.md]: This refers to the whale token right?

File: gitbook/smart-contracts/assets /denoms.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+| Chihuahua  | ibc/FA7112322CE7656DC84D441E49BAEAB9DC0AB3C7618A178A212CDE8B3F17C70B             |   [![WHALE logo](https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/white-whale.png)](https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/white-whale.png)    |
+| Injective  | IBC/D6E6A20ABDD600742D22464340A7701558027759CE14D12590F8EA869CCCF445             |   [![WHALE logo](https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/white-whale.png)](https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/white-whale.png)    |
+| Migaloo    | uwhale                                                                             |   [![WHALE logo](https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/white-whale.png)](https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/white-whale.png)    |
```
Comment (kerber0x) [inline gitbook/smart-contracts/assets /denoms.md:12]: Terra is missing sir `ibc/36A02FFC4E74DF4F64305130C3DFA1B06BEAC775648927AA44467C76A77AB8DB`

**Review by kerber0x — CHANGES_REQUESTED** [review]
Awesome addition, here are some improvements though

**Comment by 0xFable** [comment]
Thank you for review @kerber0x, just noticed it when opening another PR. I took care of and pushed the requested changes there 🙏

**Review by kerber0x — APPROVED** [review]
Lovely, well done! very useful indeed!

**Pull request merged**